spec/index.bs

```diff
@@ -1074,6 +1075,8 @@ We define <dfn dfn>subresource</dfn> to be either a whole buffer, or a [=texture
 
     - Each usage in |U| is [=internal usage/input=], [=internal usage/constant=], [=internal usage/storage-read=], or [=internal usage/attachment-read=].
     - Each usage in |U| is [=internal usage/storage=].
+        Multiple such usages are allowed even though they are writable;
+        this is the <dfn dfn>usage scope storage exception</dfn>.
     - |U| contains exactly one element: [=internal usage/attachment=].
```

Comment:
[1] here this would say Each usage in U is attachment . With a note saying that using as multiple attachment at the same time is forbidden through a different rule. (like the note for the usage scope storage exception)